sematic/api/endpoints/runs.py

```diff
@@ -101,6 +101,17 @@ def list_runs_endpoint(user: Optional[User]) -> flask.Response:
     except ValueError as e:
         return jsonify_error(str(e), HTTPStatus.BAD_REQUEST)
 
+    search_string, search_predicates = flask.request.args.get("search"), None
+    if search_string is not None:
+        search_predicates = sqlalchemy.or_(
+            Run.name.ilike(f"%{search_string}%"),
+            Run.calculator_path.ilike(f"%{search_string}%"),
+            Run.description.ilike(f"%{search_string}%"),
+            Run.source_code.ilike(f"%{search_string}%"),
+            Run.id.ilike(f"%{search_string}%"),
+            Run.tags.ilike(f"%{search_string}%"),
```

Comment:
Since the DB column type is `text` I suppose it will work the same way as an ordinary string [ilike](https://www.postgresql.org/docs/current/functions-matching.html) match.

For the part of the test. Yes, will do that.